packages/mui-base/src/utils/index.ts

```diff
@@ -4,4 +4,5 @@ export { default as extractEventHandlers } from './extractEventHandlers';
 export { default as isHostComponent } from './isHostComponent';
 export { default as resolveComponentProps } from './resolveComponentProps';
 export { default as useSlotProps } from './useSlotProps';
+export { default as unstable_mergeSlotProps } from './mergeSlotProps';
```

Comment:
I don't think we need to add `unstable_` prefix when exporting from the `utils` subdirectory.

Reply:
Fixed.